Documentation/contributing.md

```diff
@@ -10,7 +10,7 @@ Consistent code conventions are important for several reasons:
 * To make it easy to debug the code, with both a system call tracer and GNU debuggers. It should be easy to set breakpoints, view locals, and display and view data structures.
 * To attempt to improve code quality through consistency, and requiring patterns that are less likely to result in bugs either initially, or after code modification.
 
-For more information, please refer to [coding-convention.md](Documentation/coding-convention.md).
+For more information, please refer to [coding-convention.md](coding-convention.md).
```

Comment:
To be compatible with `github.io`
```suggestion
For more information, please refer to [coding-convention.md](https://github.com/nnstreamer/nntrainer/blob/main/CONTRIBUTING.md).
```

Reply:
nntrainer --> nnstreamer?? Also, this change makes a link to `github.com` from `github.io`.

Reply:
> nntrainer --> nnstreamer?? 
Right. 
```suggestion
For more information, please refer to [coding-convention.md](https://github.com/nnstreamer/nnstreamer/blob/main/CONTRIBUTING.md).
```
> Also, this change makes a link to github.com from github.io.
Right. But ㅑn the current archive, this is the only way not to destroy links on both sides.

Reply:
Since the `coding-convention.md` is in the same directory with `contributing.md`, I think
```markdown
[coding-convention.md](coding-convention.md).
```
won't break both `github.com` and `github.io`.

From the `meson`'s document mentioned above, the files in the same directory are just linked with using the file name.
```markdown
# Built-in options

Meson provides two kinds of options: [build options provided by the build files](Build-options.md) and built-in options that are either universal options, base options, compiler options.
```

Reply:
No. It will break the link in `github.io` with this change.
It is not important whether it is in the same directory, but how it is defined in sitemap (Documentation/gotdoc/sitemap.txt).

Reply:
You can build the html page as follows:
${NNST_ROOT}/Documentation/hotdoc/gen-hotdoc.sh

Reply:
Oh, I see. Thank you for your kind explanation!